docs/changes/4.1.0.rst

```diff
@@ -14,9 +14,21 @@ on GitHub.
 
 .. NOTE(vytas): No changes to the supported platforms (yet).
 
-
 .. towncrier release notes start
 
+
+Misc
```

Comment:
Please do not render out newsfragments yourself; a Falcon release manager does that when the  time comes.
Moreover, this fragment looks stale, it is already release as part of Falcon 4.0.2.